code/modules/vehicles/cars/clowncar.dm

```diff
@@ -13,6 +13,11 @@
 	var/droppingoil = FALSE
 	var/RTDcooldown = 150
 	var/lastRTDtime = 0
+	
+/obj/vehicle/sealed/car/clowncar/Initialize()
+	.=..()
+	var/datum/component/riding/D = LoadComponent(/datum/component/riding)
+	D.vehicle_move_delay = 0.6
```

Comment:
remove this. go to
https://github.com/tgstation/tgstation/blob/a45e254d548fcd11f3f811e229843ae68f0907f7/code/modules/vehicles/cars/car.dm#L13

and replace it with
var/datum/component/riding/D = LoadComponent(/datum/component/riding)
D.vehicle_move_delay = move_delay

Reply:
Yes, I wholly understand how that works. I *chose* to do it this way to be in-line with how everything else that uses the ridden datum works. Also, if you want to do other things to the datum for the vehicle, you'd have to do it here anyway. Like making the vehicle spaceworthy, or adjusting the offsets (basically irrelevant for cars), or what have you. IMO your suggestion isn't the right fix if you ever want to use the riding datum for anything besides move delay.

Reply:
we have a specific move_delay variable on the vehicles, if we change that it should affect the component. this is literaly never ever any different. it makes /zero/ sense to use some magic number in init instead of using the variable specificially existing to use to set the movement speed.

Reply:
while you're at it. set slowvalue to 0 on the component.